packages/frontend-2/pages/dashboard.vue

```diff
@@ -0,0 +1,143 @@
+  </div>
+</template>
+<script setup lang="ts">
+import { dashboardProjectsQuery } from '~~/lib/dashboard/graphql/queries'
+import type { TutorialItem, QuickStartItem } from '~~/lib/dashboard/helpers/types'
+import { useQuery } from '@vue/apollo-composable'
+import { useMixpanel } from '~~/lib/core/composables/mp'
+import GhostContentAPI from '@tryghost/content-api'
+import { docsPageUrl, forumPageUrl } from '~~/lib/common/helpers/route'
+import type { ManagerExtension } from '~~/lib/common/utils/downloadManager'
+import { downloadManager } from '~~/lib/common/utils/downloadManager'
+import { ToastNotificationType, useGlobalToast } from '~~/lib/common/composables/toast'
+
+useHead({ title: 'Dashboard' })
+
+definePageMeta({
+  middleware: ['auth']
+})
+
+const config = useRuntimeConfig()
+const mixpanel = useMixpanel()
+const { result: projectsResult } = useQuery(dashboardProjectsQuery)
+const { triggerNotification } = useGlobalToast()
+
+const ghostContentApi = new GhostContentAPI({
+  url: 'https://speckle.systems',
+  key: config.public.ghostApiKey,
+  version: 'v5.0'
+})
+
+const tutorials = ref<TutorialItem[]>([])
+const quickStartItems = shallowRef<QuickStartItem[]>([
+  {
+    title: 'Install Speckle manager',
+    description: 'Use our Manager to install and manage Connectors with ease.',
+    buttons: [
+      {
+        text: 'Download for Windows',
+        onClick: () => onDownloadManager('exe')
+      },
+      {
+        text: 'Download for Mac',
+        onClick: () => onDownloadManager('dmg')
+      }
+    ]
+  },
+  {
+    title: "Don't know where to start?",
+    description: "We'll walk you through some of most common usage scenarios.",
+    buttons: [
+      {
+        text: 'Open documentation',
+        props: { to: docsPageUrl, external: true }
+      }
+    ]
+  },
+  {
+    title: 'Have a question you need answered?',
+    description: 'Submit your question on the forum and get help from the community.',
+    buttons: [
+      {
+        text: 'Ask a question',
+        props: { to: forumPageUrl, external: true }
+      }
+    ]
+  }
+])
+
+const projects = computed(() => projectsResult.value?.activeUser?.projects.items)
+
+async function fetchTutorials() {
+  const posts = await ghostContentApi.posts.browse({
+    limit: 8,
+    filter: 'visibility:public'
+  })
+
+  tutorials.value = posts
+    .filter((post) => post.url)
+    .map((post) => ({
+      id: post.id,
+      readingTime: post.reading_time,
+      publishedAt: post.published_at,
+      url: post.url,
+      title: post.title,
+      featureImage: post.feature_image
+    }))
+}
+
+const onDownloadManager = (extension: ManagerExtension) => {
+  try {
+    downloadManager(extension)
+
+    mixpanel.track('Manager Download', {
+      os: extension === 'exe' ? 'win' : 'mac'
+    })
+  } catch {
+    triggerNotification({
+      type: ToastNotificationType.Danger,
+      title: 'Download failed'
+    })
+  }
+}
+
+onMounted(() => {
+  fetchTutorials()
```

Comment:
1. https://nuxt.com/docs/api/composables/use-async-data
2. use lazy key, if you want it  to be non blocking

Reply:
Its important to use nuxt's data fetching utils, cause they all kinds of stuff like caching, for example. Currently you will refetch the same data over and over again every time you open the dashboard, even if its already available.

Reply:
@fabis94 This will only work if we call an endpoint directly, right? It's using this Ghost API package, so I don't know if there is a way to make that work? Alternatively maybe I can switch not using their package and just call their API directly

```
const ghostContentApi = new GhostContentAPI({
  url: 'https://speckle.systems',
  key: config.public.ghostApiKey,
  version: 'v5.0'
})
```

Reply:
@Mikehrn No, useAsyncData allows you to do any kind of arbitrary async activity in it, you don't have to use their $fetch() util

Reply:
@fabis94 Nice, I didn't know that, I've updated, also added a small util to resize images returned from the Ghost API (as they seem to have no build-in option, so it always return full images which takes long time to load) :)